src/main/java/com/epam/poc/pageUIs/homepage/HomePageUI.java

```diff
@@ -4,4 +4,19 @@ public class HomePageUI {
     public static final String AO_KHOAC_LINK_XPATH = "//*[@class='header-with-search__search-section']//a[text()='Áo Khoác']";
     public static final String CLOSE_BUTTON_CLASS_NAME = "shopee-popup__close-btn";
     public static final String SHOPEE_BANNER_POPUP_TAG_NAME = "shopee-banner-popup-stateful";
+    public static final String helpButton = ".navbar__link--help";
```

Comment:
You should make the name of the variable with capital letters.
For example: "helpButton" => HELP_BUTTON_CSS.

The same for other variables